packages/xdl/src/start/ManifestHandler.ts

```diff
@@ -231,14 +231,16 @@ export async function getManifestResponseAsync({
   manifest.debuggerHost = await UrlUtils.constructDebuggerHostAsync(projectRoot, hostname);
   manifest.logUrl = await UrlUtils.constructLogUrlAsync(projectRoot, hostname);
   manifest.hostUri = await UrlUtils.constructHostUriAsync(projectRoot, hostname);
-  // Resolve all assets and set them on the manifest as URLs
-  await ProjectAssets.resolveManifestAssets({
-    projectRoot,
-    manifest: manifest as ExpoAppManifest,
-    async resolver(path) {
-      return manifest.bundleUrl!.match(/^https?:\/\/.*?\//)![0] + 'assets/' + path;
-    },
-  });
+  if (manifest.sdkVersion) {
```

Comment:
What happens in a local development project when the sdkVersion isn't used? are assets not available?

Reply:
I'm reworking this to try and include `sdkVersion` whenever possible. Namely I am doing (`sdk` OR `rtv`) instead of (`sdk` XOR `rtv`). A lot of context in #updates channel if you are curious.

What are the situations that sdkVersion would not be available? Are there any now that this PR is doing OR instead of XOR?